Documentation Standard Updates (#50)
fivetran/dbt_xero: README.md

</p>

# Xero Transformation dbt Package ([Docs](https://fivetran.github.io/dbt_xero/))
## What does this dbt package do?
- Produces modeled tables that leverage Xero data from [Fivetran's connector](https://fivetran.com/docs/applications/xero) in the format described by [this ERD](https://fivetran.com/docs/applications/xero#schemainformation) and builds off the output of our [Xero source package](https://github.com/fivetran/dbt_xero_source).

- Provides analytics-ready models, including a profit and loss report, general ledger, and balance sheet report.
- Generates a comprehensive data dictionary of your source and modeled Xero data through the [dbt docs site](https://fivetran.github.io/dbt_xero/).

> The following table provides a detailed list of all models materialized within this package by default.
> 
> | **Model**                | **Description**                                                                                      |

Author: ```suggestion
<!--section=“xero_transformation_model"-->

The following table provides a detailed list of all tables materialized within this package by default.

| **Table**                | **Description**                                                                                      |
```

| [xero__balance_sheet_report](https://github.com/fivetran/dbt_xero/blob/main/models/xero__balance_sheet_report.sql)    | Each record represents the state of the balance sheet for a given account on a given month.                            |
| [xero__invoice_line_items](https://github.com/fivetran/dbt_xero/blob/main/models/xero__invoice_line_items.sql)      | Each record represents an invoice line item enriched with the account, contact, and invoice information.                   |

> ## How do I use the dbt package?

Author: ```suggestion
<!--section-end-->

## How do I use the dbt package?
```

    xero_database: your_database_name 
```

### (Optional) Step 4: Additional configurations

> 

Reviewer: <details open><summary>Expand/Collapse configurations</summary>
<br>

> 

Reviewer: ```suggestion
<details open><summary>Expand/Collapse configurations</summary>
<br>
```

    xero_database: your_database_name 
```

### (Optional) Step 4: Additional configurations
<details open><summary>Expand/Collapse configurations</summary>

> <br>

Reviewer: ```suggestion
<br>

```

    xero_database: your_database_name 
```

### (Optional) Step 4: Additional configurations
<details open><summary>Expand/Collapse configurations</summary>

> <br>

Reviewer: ```suggestion
```

</details>

### (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™

> 

Reviewer: ```suggestion
<details><summary>Expand for details</summary>
<br>
```

</details>

### (Optional) Step 5: Orchestrate your models with Fivetran Transformations for dbt Core™
<details><summary>Expand for details</summary>

> <br>

Reviewer: ```suggestion
<br>

```
